kube-enforcer/templates/role.yaml

```diff
@@ -20,8 +20,51 @@ rules:
   - apiGroups: [ "*" ]
     resources: [ "leases" ]
     verbs: [ "get", "list", "create", "update" ]
+  - apiGroups: [ "" ]
+    resources: [ "secrets" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: [ "update", "create" ]
 {{- if eq .Values.global.platform "openshift" }}
   - apiGroups: ["*"]
     resources: ["pods"]
     verbs: ["create", "delete"]
 {{- end }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: starboard-operator
+  namespace: aqua
```

Comment:
Should this be set? is it not better to leave it not set and let Helm install it in the release namespace? we don't run things in the `aqua` namespace and this breaks our deployment.